microsoft/PowerToys
# Migrate winappdriver test project to .NET Core 3.1

## Summary
Migration of winappdriver harness project to .NET Core to facilitate overall .NET Core migration effort (#776).

## PR Checklist
* [x] Applies to #776
* [x] I've discussed this with core contributors already. If not checked, I'm ready to accept this work might be rejected in favor of a different grand plan. Issue number where discussion took place: #776

## Validation Steps

1. Build solution/project
2. Verify no build errors.

## Note
Tests continue to fail at this time. A separate PR will be created to fix test failures.
